src/components/organisms/SwitchableMode/Switch/index.js

```diff
@@ -31,8 +31,10 @@ const ModeSwitch = ({
   variant = 'switches.primary',
   left = 'GitLab',
   leftMode = 'gitlab',
-  right = 'GitHub',
-  rightMode = 'github',
+  center = 'Github',
+  centerMode = 'github',
+  right = 'Bitbucket',
+  rightMode = 'bitbucket',
```

Comment:
Just bikeshedding: should order be GitHub / GitLab / Bitbucket instead?

Reply:
I think we choose GL over GH because of GL relationship

Reply:
🤷🏼‍♂️ 👌🏼